Prometheus.NetStandard/Prometheus.NetStandard.csproj

```diff
@@ -1,9 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
```

Comment:
adding the reference assemblies package means we no longer have to condition the build based on OS.